test/runtests.py

```diff
@@ -16,6 +16,7 @@
 
 from mozillapulse import consumers, publishers
 from mozillapulse.messages.test import TestMessage
+from kombu import Exchange
```

Comment:
Third-party imports should be in their own (blank-line surrounded) section between imports from built-in modules and local imports.